Fix crash when clicking History menu with no browser window
https://github.com/QupZilla/qupzilla/issues/1487
https://github.com/QupZilla/qupzilla/commit/b6f913dafe2fce92f406ec3f21970df2151bde8d
--- src/lib/history/historymenu.cpp.orig
+++ src/lib/history/historymenu.cpp
@@ -81,7 +81,7 @@ void HistoryMenu::showHistoryManager()
 void HistoryMenu::aboutToShow()
 {
     // Set enabled states for Back/Forward actions according to current WebView
-    TabbedWebView* view = m_window->weView();
+    TabbedWebView* view = m_window ? m_window->weView() : 0;
 
     if (view) {
         actions().at(0)->setEnabled(view->history()->canGoBack());
